docs/download/download.md

```diff
@@ -0,0 +1,24 @@
+# Download
+
+# Verification
+ you can follow these [procedures](https://www.apache.org/dyn/closer.cgi#verify) and the [KEYS](https://dist.apache.org/repos/dist/dev/incubator/weex/KEYS) file to verify the download files
+
+
+# Apache Weex
+> GitHub: [https://github.com/apache/incubator-weex](https://github.com/apache/incubator-weex)    
+> Release Notes: [https://github.com/apache/incubator-weex](https://github.com/apache/incubator-weex/releases)
+
+## 0.20.0
+- [source](http://www.apache.org/dist/incubator/weex/0.20.0/apache-weex-incubating-0.20.0-src.tar.gz) | [asc](http://www.apache.org/dist/incubator/weex/0.20.0/apache-weex-incubating-0.20.0-src.tar.gz.asc) | [sha512](http://www.apache.org/dist/incubator/weex/0.20.0/apache-weex-incubating-0.20.0-src.tar.gz.sha512)

```

Comment:
* According to [apache release download page](http://www.apache.org/dev/release-download-pages#links), the sha512 and asc file should be under *https* instead of http.
* According to [download script](http://www.apache.org/dev/release-download-pages#closer), the download link for source should be under mirror address, like https://www.apache.org/dyn/closer.cgi?filename=incubator/weex/0.20.0/apache-weex-incubating-0.20.0-src.tar.gz&action=download